zulipterminal/ui_tools/buttons.py

```diff
@@ -52,22 +54,27 @@ def update_count(self, count: int) -> None:
 
     def update_widget(self, count_text: str) -> Any:
         # Note that we don't modify self._caption
+        if isinstance(self.postfix_caption, tuple):
+            postfix_text = self.postfix_caption[1]
+        else:
+            postfix_text = self.postfix_caption
         max_caption_length = (self.width_for_text_and_count -
-                              len(count_text))
+                              len(count_text) - len(postfix_text))
         if len(self._caption) > max_caption_length:
             caption = (self._caption[:max_caption_length-1] +
                        '\N{HORIZONTAL ELLIPSIS}')
         else:
             caption = self._caption
         num_extra_spaces = (
             self.width_for_text_and_count - len(count_text) - len(caption)
+            - len(postfix_text)
         )
 
         # NOTE: Generated text does not include space at end
         self._w = urwid.AttrMap(urwid.SelectableIcon(
             [' ', self.prefix_character, self.post_prefix_spacing,
              '{}{}'.format(caption, num_extra_spaces*' '),
-             ' ', ('idle',  count_text)],
+             ' ', self.postfix_caption, ('idle',  count_text)],
```

Comment:
If we ever had both a `postfix_caption` and `count_text`, then I expect we'd want a space between them, which would have to be added manually right now. However, this is fine for now.